### ISSUE TITLE
Missing API key should fail build

### ISSUE THREAD

**Issue by michaelborn** [foundeo/fixinator#9]
I noticed if the API key is not set (due to id10t errors :roll_eyes: ) the CI build still passes. It seems 403 errors from the Fixinator server are caught, and do not return a non-zero exit code.

For reference, here's my Github Actions run:

https://github.com/michaelborn/cfPlaid/runs/3692858559?check_suite_focus=true

With a brief snippet of the error:

```
Note: The enterprise version allows you to run the code scanner fully on your own servers.
Detected CI Environment, I will continue without prompting
Fixinator API Server: https://api.fixinator.app/v1/scan
["/github/workspace/models/","/github/workspace/ModuleConfig.cfc"]

 √ | Scanning /github/workspace/

---- Fixinator Client Error ----

Fixinator API Key () is invalid, disabled or over the API request limit. Please contact Foundeo Inc. for assistance. Please provide your API key in correspondance. https://foundeo.com/contact/ 
403 Forbidden {"message":"Forbidden"}
```

This error should cause a non-zero exit code to be returned so the build will fail.

**Comment by michaelborn** [comment]
It looks like the error is caught and displayed here:

https://github.com/foundeo/fixinator/blob/master/commands/fixinator.cfc#L339

should these errors `rethrow`? Or no?

**Comment by pfreitag** [comment]
Fixed in a53bc0cc89ff157c2f5b8612db8dd06e43f8ccc0 now uses `error()`
